addons/sale/wizard/mass_cancel_quotations_wizard.py

```diff
@@ -0,0 +1,31 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+class MassCancelQuotationsWizard(models.TransientModel):
+    _name = 'sale.mass.cancel.quotations'
```

Comment:
```suggestion
    _name = 'sale.mass.cancel.orders'
```
change tech names to orders, leave user-shown strings to quotations ;)